Cargo.toml

```diff
@@ -7,12 +7,14 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-gtk = { package = "gtk4", version = "0.6" }
+gtk = { package = "gtk4", version = "0.8" }
 
 [dependencies.plotters]
-git = "https://github.com/plotters-rs/plotters"
+version = "0.3.5"
 default_features = false
 features = ["surface_series"]
 
 [dependencies.plotters-cairo]
-git = "https://github.com/plotters-rs/plotters-cairo"
+version = "0.6.0"
+git = "https://github.com/sdroege/plotters-cairo"
+branch = "cairo-rs-0.19"
```

Comment:
This should be removed after merging the other PR.

Reply:
The PR is merged and the new version is published on crates.io :)

Reply:
And updated to use the version from crates.io. Thanks!